id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/constant/HandleStatusLifecycle.java

```diff
@@ -0,0 +1,9 @@
+package io.mosip.idrepository.core.constant;
+
+/**
+ * @author Ritik Jain
+ *
+ */
+public enum HandleStatusLifecycle {
+	ACTIVATED, DELETED, DELETE_REQUESTED;
```

Comment:
I think instead of DELETED it should be DELETE, 

ACTIVATED -> DELETE -> DELETE_REQUESTED

Reply:
changes done